src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -171,16 +171,33 @@
 
 (rf/reg-event-fx
  :wallet/collectible-ownership-update-finished
- (fn [{:keys [db]} [{:keys [accounts chainId]}]]
+ (fn [{:keys [db]} [{:keys [accounts chainId message]}]]
    (let [address            (first accounts)
          pending-chain-ids  (get-in db [:wallet :ui :collectibles :updating address])
          updated-chain-ids  (disj pending-chain-ids chainId)
-         all-chain-updated? (and (some? pending-chain-ids) (empty? updated-chain-ids))]
+         all-chain-updated? (and (some? pending-chain-ids) (empty? updated-chain-ids))
+         collectible-id     (data-store/rpc->collectible-id message)]
```

Comment:
I have a feeling that this might cause a bug because we are taking only the first entry. Instead, it should be a set of unique IDs where we can check if the pending collectible id is present in it and dispatch the events accordingly. WDYT?

Reply:
Sorry, I didn’t quite get the issue. I’m storing all collectible unique IDs in [:wallet :ui :collectibles :pending], so it’s not just the first entry. There can be more than one pending collectible. When here it receives the data, if it’s in the pending list, it will trigger the event to update it

Reply:
As far i saw it only give data about one collectible (added/updated/removed), do we have real example of getting information about multi collectible at one event?

Reply:
True that. It will return only one collectible data after the transaction is done. Maybe the scenario I'm thinking is a bit edge case, like during that time of fetching in status-go, if that account received any collectable from any other account (between the fetching interval), the status-go signals will include both collectible details too. Nevertheless, that is something we can do to improve later after we refactor the collectible data in app-db.

It's good for now. Thanks for the fix! @mohsen-ghafouri 🙌